source/getting-started.rst

```diff
@@ -58,26 +59,25 @@ There are 3 API endpoints for this service, refer to :ref:`reference` for
 parameter definition. Brief descriptions are:
 
 /preauth
-  This is used when performing transactions from a browser (as opposed to using an SDK), where it will
-  return an optional 3-D Secure Method URL, which is used for browser
-  fingerprinting. This can support risk-based analysis and assist in ensuring
-  a flow where the cardholder is not challenged.
+  This is used when performing transactions from a browser (as opposed to using
+  an SDK), where it will return an optional 3-D Secure Method URL, which is
```

Comment:
```suggestion
  This is used when performing transactions from a browser as opposed to using
  an SDK. It will return an optional 3-D Secure Method URL, which is
```

Reply:
:+1: